blocks/Xcos/blocks/CLOCK_c.py

```diff
@@ -3,6 +3,17 @@
 from blocks.SplitBlock import SplitBlock
 from common.AAAAAA import *
 
+block_id = ['128c18ea:1383ab8277d:-748d', '-73e75f0:167968eb73f:-7955',
+            '-73e75f0:167968eb73f:-7953', '-73e75f0:167968eb73f:-7950']
+link_id = ['-63efee48:189fd5ed04e:-73d5', '-63efee48:189fd5ed04e:-73d4',
+           '-63efee48:189fd5ed04e:-73d3', '128c18ea:1383ab8277e:-748d'
```

Comment:
```suggestion
           '-63efee48:189fd5ed04e:-73d3'
```
Remove from here. Already in block_id[0]